src/scss/components/_inputs.scss

```diff
@@ -76,15 +76,19 @@ $input-select-margin-right: 1;
   border-radius: 50%;
 }
 
-.usa-radio__input:disabled + .usa-radio__label::before,
```

Comment:
Should we want this to apply to radio buttons as well? As implemented, the border of the unselected, disabled radio is thicker than the unselected, enabled radio and lighter than the disabled checkbox.

![image](https://user-images.githubusercontent.com/1779930/168663938-04ef2ef2-d414-42cd-b45a-a853334b2589.png)

![image](https://user-images.githubusercontent.com/1779930/168663954-9411bda6-1281-4adc-9303-ed0fddf1e434.png)

![image](https://user-images.githubusercontent.com/1779930/168663977-d399da82-aec4-43a7-b3ea-ba5e98472ef4.png)

Keeping this selector makes it look a bit more expected:

![image](https://user-images.githubusercontent.com/1779930/168664100-2400082f-4c7e-45a9-a3c2-8d0e27a40dea.png)